flink-end-to-end-perf-tests/flink-basic-operations/src/main/java/org/apache/flink/basic/utils/WordCountData.java

```diff
@@ -0,0 +1,91 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.basic.utils;
+
+import java.util.Random;
+
+/**
+ * Provides the default data sets used for the WordCount example program. The default data sets are used, if no
+ * parameters are given to the program.
+ */
+public class WordCountData {
+
+	WordCountData() {}
+
+	private static final String chars = "abcdefghijklmnopqrstuvwxyz";
+	private static final String[] words = new String[] {
+		"To be, or not to be,--that is the question:--",
+		"Whether 'tis nobler in the mind to suffer",
+		"The slings and arrows of outrageous fortune",
+		"Or to take arms against a sea of troubles,",
+		"And by opposing end them?--To die,--to sleep,--",
+		"No more; and by a sleep to say we end",
+		"The heartache, and the thousand natural shocks",
+		"That flesh is heir to,--'tis a consummation",
+		"Devoutly to be wish'd. To die,--to sleep;--",
+		"To sleep! perchance to dream:--ay, there's the rub;",
+		"For in that sleep of death what dreams may come,",
+		"When we have shuffled off this mortal coil,",
+		"Must give us pause: there's the respect",
+		"That makes calamity of so long life;",
+		"For who would bear the whips and scorns of time,",
+		"The oppressor's wrong, the proud man's contumely,",
+		"The pangs of despis'd love, the law's delay,",
+		"The insolence of office, and the spurns",
+		"That patient merit of the unworthy takes,",
+		"When he himself might his quietus make",
+		"With a bare bodkin? who would these fardels bear,",
+		"To grunt and sweat under a weary life,",
+		"But that the dread of something after death,--",
+		"The undiscover'd country, from whose bourn",
+		"No traveller returns,--puzzles the will,",
+		"And makes us rather bear those ills we have",
+		"Than fly to others that we know not of?",
+		"Thus conscience does make cowards of us all;",
+		"And thus the native hue of resolution",
+		"Is sicklied o'er with the pale cast of thought;",
+		"And enterprises of great pith and moment,",
+		"With this regard, their currents turn awry,",
+		"And lose the name of action.--Soft you now!",
+		"The fair Ophelia!--Nymph, in thy orisons",
+		"Be all my sins remember'd."
+	};
+
+	private static String[] generateData(String[] newWords, int wordLength, int seed) {
+		Random rand = new Random(seed);
+		for (int i = 0; i < newWords.length; i++) {
+			StringBuilder str = new StringBuilder();
+			for (int j = 0; j < wordLength; j++) {
+				str.append(chars.charAt(rand.nextInt(Math.min(26, wordLength))));
+			}
+			newWords[i] = str.toString();
+		}
+		return newWords;
+	}
+
+	public static String[] getWords(int wordLength, Boolean random, int seed) {
```

Comment:
In 1 of 2 cases, 2 out of 3 method arguments are unused. Better to split this into dedicated methods.

Reply:
yes, i updated